sound/hybrid_font.h

```diff
@@ -21,6 +21,8 @@ class FontConfigFile : public ConfigFile {
     CONFIG_VARIABLE2(ProffieOSBlastImageDuration, 1000.0f);
     CONFIG_VARIABLE2(ProffieOSClashImageDuration, 500.0f);
     CONFIG_VARIABLE2(ProffieOSForceImageDuration, 1000.0f);
+    CONFIG_VARIABLE2(ProffieOSMinSwingAcceleration, 0.0f);
```

Comment:
This doesn't belong in this PR

Reply:
strange, maybe I branched wrong version, will resubmit.